pom.xml

```
@@ -101,7 +101,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git-client</artifactId>
-      <version>3.4.2</version>
+      <version>3.5.1-SNAPSHOT</version>
```

Use the incremental build from git client plugin CI job.
```suggestion
      <version>3.5.1-rc2571.b85fc89d5173</version>
```